### PR TITLE
Add Invoice and treatments tables

### PR DESCRIPTION
**Pull request by edgardsoza**
Create a file named schema_based_on_diagram.sql. Done
Join tables from many-to-many relationships might not appear in the diagram. Done
Remember to add the FK indexes. Done

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -8,5 +8,28 @@ CREATE TABLE medical_histories (
...
+    payed_at TIMESTAMP,
+    FOREIGN KEY medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments (
+    FOREIGN KEY id INT REFERENCES medical_histories(id),
+    type VARCHAR(50) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
+
```
Comment (1GORDON) [inline schema_based_on_diagram.sql:31]: Great job team🤝

- [ ] As stated in the project requirements, kindly ensure all "join table" for the many-to-many relationships is created, even if they are not shown on the diagram. You may create a table between `medical_histories` and `treatments`.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -8,5 +8,28 @@ CREATE TABLE medical_histories (
...
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    FOREIGN KEY medical_history_id INT REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments (
+    FOREIGN KEY id INT REFERENCES medical_histories(id),
+    type VARCHAR(50) NOT NULL,
+    name VARCHAR(50) NOT NULL
+);
```
Comment (1GORDON) [inline schema_based_on_diagram.sql]: Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all tables created have primary keys.

File: schema_based_on_diagram.sql [hunk 3]
```diff
@@ -8,5 +8,28 @@ CREATE TABLE medical_histories (
...
+
+CREATE TABLE invoice_items (
+    id SERIAL PRIMARY KEY,
+    unit_price DECIMAL(10,2) NOT NULL,
+    quantity INT NOT NULL,
+    total_price DECIMAL(10,2),
+    FOREIGN KEY invoice_id INT REFERENCES invoices(id),
+    FOREIGN KEY treatment_id INT REFERENCES treatments(id)
```
Comment (1GORDON) [inline schema_based_on_diagram.sql:38]: Great job team 🤝

- [ ] As stated in the project requirements, kindly ensure all FOREIGN KEYS have indexes created.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @edgardsoza & @sntgopl ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (sntgopl) [reply schema_based_on_diagram.sql:31]: Thank you for your comment. We have created a table linking the treatments to each medical history and vice-versa.

File: schema_based_on_diagram.sql (hunk 3 above)
Comment (sntgopl) [reply schema_based_on_diagram.sql:38]: Hello @1GORDON. We have been checking the changes requested, and we ensured that all tables have their respective 'id' set as 'SERIAL PRIMARY KEY'. Is that what you referred to with 'indexes created'? Or should we 'CREATE INDEX' for future queries regarding the use of FOREIGN KEYS and PRIMARY KEYS?

**Review by IngDavidTM — APPROVED** [review]
Hi Team 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 
- The new tables were created in `schema_based_on_diagram.sql` file ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**